mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/BaseActivity.kt

```diff
@@ -18,13 +18,17 @@ import rs.readahead.washington.mobile.R
 import rs.readahead.washington.mobile.data.sharedpref.Preferences
 import rs.readahead.washington.mobile.util.LocaleManager
 import rs.readahead.washington.mobile.util.LockTimeoutManager
+import rs.readahead.washington.mobile.util.isScreenReaderOn
+import rs.readahead.washington.mobile.util.setupForAccessibility
 
 abstract class BaseActivity : AppCompatActivity() {
     var isManualOrientation = false
     private lateinit var container: ViewGroup
     private lateinit var loading: View
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        if (this.isScreenReaderOn())
+            supportFragmentManager.setupForAccessibility()
 
```

Comment:
can you create a separate function maybe `setupAccessibility()` and call it later in the create function, it will make the code more structured.